src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -126,6 +126,10 @@ export const validateIfHasEvidence = (
 
 export const hasNewPtsdDisability = formData =>
   some(_.get('newDisabilities', formData, []), item => {
+    if (!_.get('view:newDisabilities', formData, false)) {
```

Comment:
If `view:newDisabilities` is false, will `newDisabilities` be more than an empty array? I'm not sure how this works if they enter some information only to go back and select no to the "do you have any new disabilities" question.

Reply:
Yeah, it'll persist, which is why this is necessary.

Reply:
Makes sense. Good call. :+1: